portfolio/src/main/webapp/script.js

```diff
@@ -317,19 +325,24 @@ function initMap() {
           else {
             correctAnswerDOM(false);
 
-            var cross_mark_marker = new google.maps.Marker({
+            let cross_mark_marker = new google.maps.Marker({
```

Comment:
nit: These should be `const` if they aren't being reassigned later. It helps prevent accidentally overwriting the value in future changes.

Reply:
Ok